pages/[username]/calendar.js

```diff
@@ -1,7 +1,23 @@
-// Calendar Widget
+// Calendar Widget //
 
-export default function Calendar() {
+//import render Calendar component
+
+import Head from 'next/head'
+import Calendar from '../../components/Calendar/Calendar.js'
+
+
+//use username from GET request, current is just mock data
+let username = 'Matt';
+
+
+export default function calendar () {
+//import user state (mentor/mentee)
   return (
-    <div className='pageData'>My Calendar</div>
+    <>
+      <Head>
+          <title>{`${username}'s Calendar`}</title>
+      </Head>
+      <Calendar />
```

Comment:
I sent a message about this sunday night, but we should try to include as much logic for our component as we can in the /pages/whatever_component_you_have. This is so we can take advantage of next js server side rendering. For example, most of the <Calendar /> logic should be in this file. However, for your cell components, those can live in the components directory and be imported and used here. DM me if you need more clarification